geoportal/c2cgeoportal_geoportal/views/proxy.py

```diff
@@ -102,6 +102,13 @@ def _proxy(self, url, params=None, method=None, cache=False, body=None, headers=
             headers["Forwarded"] = ",".join([headers["Forwarded"], forwarded_str])
         else:
             headers["Forwarded"] = forwarded_str
+        # Set alternative "X-Forwarded" headers
+        for forwarded_elements in headers["Forwarded"].split(","):
```

Comment:
```suggestion
        for forwarded_elements in reversed(headers["Forwarded"].split(",")):
```
For me, currently you are reversing the order, is it wanted?

Reply:
Nope ! you are right !